.github/workflows/checks.yml

```diff
@@ -0,0 +1,20 @@
+name: checks
+on:
+  push:
+
+jobs:
+  required-checks:
```

Comment:
@hunyadi up to you but definitely could update the repo setting to make this a check that must pass for all PRs before they could be merged.

Reply:
This could also be beneficial for my PR to run the checks before [publishing to pypi](https://github.com/hunyadi/md2conf/pull/51/files#diff-40a8e37be878e2b8c149cbae07d713e9b42743acfaac10c4a266bd7ded764019R1) ( if it gets merged )